src/main/contextMenu.ts

```diff
@@ -52,7 +52,7 @@ export default class ContextMenu {
     reload = () => {
         this.dispose();
 
-        const options = {window: this.view.webContents, ...this.menuOptions};
```

Comment:
What other areas that use context menu could this affect so we can keep in mind when we test this fix?

Reply:
It's anywhere that you right-click in the app and you get that little menu. This includes right-clicking on links and such.

Reply:
@DHaussermann heads up that we want to also smoke test other context menu to make sure there are no other issues.